### PR TITLE
remove segment

### PR DESCRIPTION
**Pull request by jowlee**
### Description
Replaces segment with ampitude
Adds adroll as well for page tracking

### How Has This Been Tested?
Ran against staging and saw the logs in amplitude

### How will this change be monitored?
amplitude logs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "@reduxjs/toolkit": "1.3.4",
     "@sentry/browser": "5.15.4",
     "@walletconnect/web3-provider": "1.4.1",
+    "amplitude-js": "^8.10.0",
```
Comment (sliptype) [inline package.json:17]: Let's pin this dependency

Trying to pin all deps moving forward to reduce probability of supply chain attacks

File: public/index.html [hunk 2]
```diff
@@ -107,58 +107,41 @@
...
+    <!-- start Adroll -->
+    <script type="text/javascript">
+      const adroll_adv_id = '%REACT_APP_ADROLL_AVD_ID%';
+      const adroll_pix_id = '%REACT_APP_ADROLL_PIX_ID%';
+      const adroll_version = "2.0";
+      console.log({
+        adroll_adv_id,
+        adroll_pix_id
+      })
+      // if (nodeEnv === 'production' && reactEnv === 'production' && !adroll_adv_id.includes('%') && !adroll_pix_id.includes('%')) {
```
Comment (sliptype) [inline public/index.html:119]: Is this comment part of Adroll? weird lol

File: src/types/amplitude-js/index.d.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+declare module 'amplitude-js'
```
Comment (sliptype) [inline src/types/amplitude-js/index.d.ts:1]: I think you can add a `@types/amplitude-js` dependency instead of declaring the types manually

**Review by sliptype — APPROVED** [review]
🔥🔥🔥

File: public/index.html (hunk 2 above)
Comment (jowlee) [reply public/index.html:119]: no it's something we add to only init an analytics library if we're in production

Comment (jowlee) [reply public/index.html:119]: I need to uncomment it out, thx

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/jowlee-remove-segment

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/jowlee-remove-segment

**Pull request merged**